### PR TITLE
Clevertap alias call support

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Added support for alias call in Clevertap

### CODE REVIEW

File: docs/destinations/streaming-destinations/clevertap.mdx [hunk 1]
```diff
@@ -819,3 +819,12 @@ A sample regulation request body for deleting a user in CleverTap is shown below
   }]
 }
 ```
+## Alias
+
+The <Link to="/event-spec/standard-events/alias/">`alias`</Link> call call lets you merge different identities of a known user.
+
+A sample `alias` call is shown below:
+
+```javascript
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/clevertap.mdx:828]: @yashasvibajpai Please confirm if this sample code works.

Comment (yashasvibajpai) [reply docs/destinations/streaming-destinations/clevertap.mdx:828]: Yes. This is suitable.

File: docs/destinations/streaming-destinations/clevertap.mdx [hunk 2]
```diff
@@ -819,3 +819,12 @@ A sample regulation request body for deleting a user in CleverTap is shown below
   }]
 }
 ```
+## Alias
+
+The <Link to="/event-spec/standard-events/alias/">`alias`</Link> call call lets you merge different identities of a known user.
```
Comment (yashasvibajpai) [inline docs/destinations/streaming-destinations/clevertap.mdx:824]:
```suggestion
The <Link to="/event-spec/standard-events/alias/">`alias`</Link> call lets you merge different identities of a known user.
```
the call word got repeated.

Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/clevertap.mdx:824]: my bad!

Comment (yashasvibajpai) [reply docs/destinations/streaming-destinations/clevertap.mdx]: np 👍

**Review by yashasvibajpai — APPROVED** [review]
awesome!

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**